SlackAPI/Attachment.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace SlackAPI
+﻿using System.Collections.Generic;
```

Comment:
Why was this using added but no code was changed that would indicate this is needed?